vcpkg/get_started/setup-vcpkg.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
It might be best to just pull this file out into a separate PR. We will want to make it applicable to all 3 operating systems using tabs for the user to swap between them. It will need a bit of work so it's easier to review it that way.